SMLHelper/Initializer.cs

```diff
@@ -5,10 +5,16 @@
     using Harmony;
     using Patchers;
 
+    /// <summary>
+    /// WARNING: This class is for use only by QModManager.
+    /// </summary>
     public class Initializer
```

Comment:
Then why is it public?

Reply:
Isn't it necessary for it to be public so it can be accessed by QModManager?

Reply:
Not if i remember correctly, let me double check

Reply:
Apparently it has to be public. 

_To update in the QMM dev branch_